Summers_James/detail.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="./catalog/templates/app_base.htm" />
+
+
+
+<%block name="content">
+<div class="persCata">
+    <h1 style="left:45%; position: relative; top:18px;">${product.name}</h1>
+    <div style = "position:absolute; top: 70px; left: 20px;">
+        <ul style="display:flex; flex-direction: column; list-style: none;">
+        %for counter, im in enumerate(images):
+            %if "/static/catalog/media/products/" + im.filename != product.image_url():
```

Comment:
don't think this is needed

Reply:
hmmmm, I think you are right about this.